docs/get-started/install.mdx

```diff
@@ -181,11 +181,17 @@ There are some noteworthy items here:
 
     {/* prettier-ignore-end */}
 
-    Update your Storybook configuration file (i.e., `.storybook/main.js|ts`) to include it.
+    <Callout variant="info">
+
+      The CLI's [`add`](../api/cli-options.mdx#add) command automates the addon's installation and setup. To install it manually, see our [documentation](../addons/install-addons.mdx#manual-installation) on how to install addons.
+    
+    </Callout>
 
+    Update your Storybook configuration file (i.e., `.storybook/main.js|ts`) to enable support for this format.
+  
     {/* prettier-ignore-start */}
 
-    <CodeSnippets path="main-config-csf-addon-register.md" />
+    <CodeSnippets path="main-config-svelte-csf-register.md" />
 
     {/* prettier-ignore-end */}
```

Comment:
This is redundant with what is available at [`/addons/install-addons.mdx#manual-installation`](https://storybook.js.org/docs/addons/install-addons#manual-installation).

I do not think we should set a precedent for including this every time we document the `add` command, as that will balloon maintenance efforts.